json-rpc/types/src/views.rs

```diff
@@ -464,6 +464,18 @@ pub struct TransactionView {
     pub vm_status: VMStatusView,
     pub gas_used: u64,
 }
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+pub struct TransactionsWithProofsView {
+    pub first_transaction_version: Option<u64>,
+    pub transactions_blobs: Vec<BytesView>,
+    pub proofs: TransactionsProofsView,
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub struct TransactionsProofsView {
+    pub ledger_info_to_transaction_infos_proof: BytesView,
+    pub transaction_infos: BytesView,
```

Comment:
same here, are these LCS serialized transaction infos? If so then perhaps rename to `serialized_transaction_infos`?